src/tools/sfdk/command.cpp

```diff
@@ -453,27 +455,73 @@ class BuildEnginePropertiesAccessor : public PropertiesAccessor
         , m_vmAccessor(std::make_unique<VirtualMachinePropertiesAccessor>(
                     engine->virtualMachine()))
     {
+        m_hostName = Sdk::effectiveBuildHostName();
     }
 
     QMap<QString, QString> get() const override
     {
-        return m_vmAccessor->get();
+        return m_vmAccessor->get().unite(getOther());
     }
 
     PrepareResult prepareSet(const QString &name, const QString &value, bool *needsVmOff,
             QString *errorString) override
     {
-        return m_vmAccessor->prepareSet(name, value, needsVmOff, errorString);
+        if (auto result = m_vmAccessor->prepareSet(name, value, needsVmOff, errorString))
+            return result;
+        if (auto result = prepareSetOther(name, value, needsVmOff, errorString))
+            return result;
+        return Ignored;
     }
 
     bool set() override
     {
-        return m_vmAccessor->set();
+        return m_vmAccessor->set() && setOther();
+    }
+
+private:
+    QMap<QString, QString> getOther() const
```

Comment:
Now this is a weird name for a method. What other? Looks like m_hostName is the only "other" that this methods gets? What's the whole purpose of this method?

Reply:
Really, it should be "others" - there is none now but more to be added in future. Will fix.

Reply:
s/none/one/